promotions/lib/controllers/backend/solidus_promotions/admin/base_controller.rb

```diff
@@ -0,0 +1,58 @@
+# frozen_string_literal: true
+
+module SolidusPromotions
+  module Admin
+    class BaseController < Spree::Admin::ResourceController
```

Comment:
Investigate if this is still needed, seems like we are not using it.

Reply:
It's still needed: We use it in all resource-y controllers, and it defines the routes proxy and model name. A lot of the methods could go though!